autoload/vista/cursor.vim

```diff
@@ -203,6 +208,48 @@ function! vista#cursor#ShowDetailWithDelay() abort
         \ )
 endfunction
 
+" Show the detail of current tag/symbol under cursor.
+function! vista#cursor#FindDetail(_timer) abort
+  if empty(getline('.'))
+        \ || !exists('g:vista')
+        \ || !has_key(g:vista, 'functions')
+        \ || !has_key(g:vista, 'source')
+    call setbufvar(g:vista.source.bufnr, 'vista_cursor_info', {})
+    return
+  endif
+
+  let found = vista#util#BinarySearch(g:vista.raw, line('.'), 'line', '')
```

Comment:
I realized that we actually don't need the further processing, just providing the `found` should be enough, which contains everything, it's up to the API user how to use it, namely `call setbufvar(g:vista.source.bufnr, 'vista_cursor_info', found)`.